engine/src/main/kotlin/io/kotless/gen/factory/route/static/StaticRouteFactory.kt

```diff
@@ -77,9 +76,10 @@ object StaticRouteFactory : GenerationFactory<Application.ApiGateway.StaticRoute
             status_code = "200"
             responseParameters(mapOf("method.response.header.Content-Type" to "integration.response.header.Content-Type",
                 "method.response.header.Content-Length" to "integration.response.header.Content-Length"))
+            if(context.schema.statics[entity.resource]!!.mime.isBinary)
```

Comment:
A bit of reformatting would be good here :) Should be `if (...)`, use `Reformat file` button :)